ui/lib/config-ui/addon/components/login-settings/page/details.hbs

```diff
@@ -0,0 +1,45 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<PageHeader as |p|>
+
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{@rule.data.name}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<nav class="tabs" aria-label="navigation for rule details">
+  <ul>
+    <li>
+      <LinkTo @route="login-settings.rule.details" @model={{@rule.data}} data-test-rule-tab>
+        Details
+      </LinkTo>
+    </li>
+  </ul>
+</nav>
+
+<Toolbar />
+{{#each-in @rule.data as |key value|}}
+  {{#if (eq key "default_auth_type")}}
+    <InfoTableRow @alwaysRender={{true}} @defaultShown={{true}} @label={{capitalize "default method"}} @value={{value}} />
```

Comment:
According to the [docs](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/info-table-row.js#L24), `@defaultShown` should be a string.

Reply:
Also, for the label, you don't need the helper, just capitalize the text. And double check me on this, but it's a label not a title so it's sentence-cased. I realize we are not consistent with this in the codebase, so check with design.
```suggestion
    <InfoTableRow @alwaysRender={{true}} @defaultShown={{true}} @label="Default method" @value={{value}} />
```